tests/test_routes.py

```diff
@@ -26,6 +27,9 @@
 FLAG = False
 
 
+def hash_passowrd(password):
```

Comment:
Please correct the typo. 
If there is an in-built function with the same name, you can use hash_password_util

Reply:
Done